install.sh

```diff
@@ -18,6 +19,7 @@ cp -f $X_SOURCE_PATH/build/release/diec                             $X_PREFIX/bi
 cp -f $X_SOURCE_PATH/build/release/diel                             $X_PREFIX/bin/
 cp -f $X_SOURCE_PATH/LINUX/io.github.horsicq.detect-it-easy.desktop $X_PREFIX/share/applications/
 cp -Rf $X_SOURCE_PATH/LINUX/hicolor/                                $X_PREFIX/share/icons/
+cp -f $X_SOURCE_PATH/LINUX/io.github.horsicq.detect-it-easy.metainfo.xml $X_PREFIX/share/metainfo/
```

Comment:
This change messes up the indentation of this section...

If you think that doesn't look good, let me know and I'll add extra whitespaces to the lines above to align the two columns.